pages/onboarding.html

```
@@ -15,9 +15,11 @@
     <div class="stage" id="stage">
       <section class="panel active" data-index="0">
         <div class="text">
-          <h2>Welcome to the club</h2>
-          <p>Welcome to our alliance, and congratulations for passing through the application stage! This article serves as an
+            <h2>Welcome to the club</h2>
+            <p>Welcome to our alliance, and congratulations for passing through the application stage! This article serves as an
   orientation on your next steps, as well as some important things to take note as a member wiki.</p>
+            <p>As a member wiki, you are expected to maintain a certain standard of quality and activity. This is because your wiki is not only representing itself, but also the alliance as a whole, and we want to make sure that the alliance is associated with good quality wikis. We also want to make sure that your wiki is in a good position to benefit from being in the alliance and to contribute to the alliance.</p>
+            <p>To help maintain this standard, we have a few requirements for our member wikis, which can be found on our <a href="{{ "/policies" | relative_url }}">Policies page</a>. Do make sure to read through them and understand them well. If you have any questions about them, feel free to ask in our Discord server!</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)